### PR TITLE
SLVUU-102 simplify getApplicationLayout response

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
### Changes
- GET /application-layout to return JSONLayout type (omit username)
- Delete ApplicationLayoutDTO (server side)
- Delete ApplicationLayout type (client side)

### CODE REVIEW

**Review by pling-scottlogic — APPROVED** [review]

**Review by pling-scottlogic — CHANGES_REQUESTED** [review]
All the changes look good. Just need to change the target branch to main.

**Comment by pling-scottlogic** [comment]
This might just be me, but I don't like the 256 commits showing up. You could tidy this up by doing a soft reset to main, then committing the resulting changes and force pushing.

File: layout-server/src/test/java/org/finos/vuu/layoutserver/controller/ApplicationLayoutControllerTest.java [hunk 1]
```diff
@@ -27,17 +27,16 @@ public void setup() {
     }
 
     @Test
-    public void getApplicationLayout_anyUsername_returnsLayoutFromService() throws JsonProcessingException {
+    public void getApplicationLayout_anyUsername_returnsLayoutDefinitionFromService() throws JsonProcessingException {
```
Comment (cfisher-scottlogic) [inline layout-server/src/test/java/org/finos/vuu/layoutserver/controller/ApplicationLayoutControllerTest.java:30]: My IDE is pointing out that the throws list is no longer required as `JsonProcessingException` isn't thrown.

File: layout-server/src/test/java/org/finos/vuu/layoutserver/service/ApplicationLayoutServiceTest.java [hunk 2]
```diff
@@ -39,13 +39,12 @@ public void setup() {
     public void getApplicationLayout_noLayout_returnsDefault() throws JsonProcessingException {
```
Comment (cfisher-scottlogic) [inline layout-server/src/test/java/org/finos/vuu/layoutserver/service/ApplicationLayoutServiceTest.java:39]: Same comment about `JsonProcessingException` as above.

**Review by cfisher-scottlogic — COMMENTED** [review]
Just a few minor observations.

**Review by pling-scottlogic — APPROVED** [review]

File: layout-server/src/test/java/org/finos/vuu/layoutserver/controller/ApplicationLayoutControllerTest.java (hunk 1 above)
Comment (vferraro-scottlogic) [reply layout-server/src/test/java/org/finos/vuu/layoutserver/controller/ApplicationLayoutControllerTest.java]: done :)

File: layout-server/src/test/java/org/finos/vuu/layoutserver/service/ApplicationLayoutServiceTest.java (hunk 2 above)
Comment (vferraro-scottlogic) [reply layout-server/src/test/java/org/finos/vuu/layoutserver/service/ApplicationLayoutServiceTest.java:39]: done :)

**Review by cfisher-scottlogic — APPROVED** [review]

**Pull request closed without merging**